src/App.scss

```diff
@@ -140,6 +140,28 @@
 .ms-Icon {
   font-size: 1em;
 }
+.icon-2x {
+    font-size: 2em;
+}
+.icon-3x {
+    font-size: 3em;
+}
+
+.icon-4x {
+    font-size: 4em;
+}
+.icon-5x {
+    font-size: 5em;
+}
+.icon-6x {
+    font-size: 6em;
+}
+.icon-7x {
+    font-size: 7em;
+}
+.icon-8x {
+    font-size: 8em;
+}
```

Comment:
those classes been added, but never used

Reply:
icon-7x used in homePage